### PR TITLE
38 Get Professors frontend

### PR DESCRIPTION
**Pull request by ashahrod**
Added the getProfessorsMatching function. All profs that match filters, if any, are returned.

### CODE REVIEW

File: frontend/src/App.js [hunk 1]
```diff
@@ -36,6 +36,17 @@ function App(props){
...
 
+   function getProfessorsMatching(req){
+      const prof_matching = [];
+      const response = axios.get("http://localhost:5001/api/professor", { params: { name: req.body } })
+      // for over all the indices of the response table, and get each name and add it to the prof_matching[]
+      for(let i = 0; i < response.length; i++){
+         prof_matching.push(response[i].name);
+      }
+      // const names = response.data.name
+
```
Comment (wes-curl) [inline frontend/src/App.js:47]: Does this function return the professors that match? It's supposed to, I belive.

Comment (ashahrod) [reply frontend/src/App.js:47]: Yes it does, just finished testing!

**Review by wes-curl — APPROVED** [review]
All good, edits complete

**Review by sajanjohal — APPROVED** [review]
Looks good to me!

**Pull request merged**